nuclear_simulator/data_gen/runners/maintenance_scenario_runner.py

```
@@ -0,0 +1,1498 @@
+            maintenance_history = state_manager.get_maintenance_history()
+            data_point['maintenance_history_count'] = len(maintenance_history)
+            
+            # Add component health snapshots for key components
+            key_components = ['FW-001', 'TB-001', 'SG-001', 'CD-001']
+            for component_id in key_components:
+                try:
+                    snapshot = state_manager.get_component_state_snapshot(component_id)
+                    for param, value in snapshot.items():
+                        if isinstance(value, (int, float)):
+                            data_point[f"{component_id}_{param}"] = value
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)